### ISSUE TITLE
Lint does not pick missing hash curly braces in epp arguments

### ISSUE THREAD

**Issue by yvf** [puppetlabs/puppet-lint#248]
## Describe the Bug
In the below code, the enclosing `{}` are missing. Puppet parses it without error (and it works !) but AFAICS this is totally undocumented behavior, and is rather confusing. I think this should be a lint error or warning.

```
class test_epp_hash {
  file {
    'test.file':
      content => epp('template.epp',
        'key' => 'value',
      );
  }
}
```
## Expected Behavior
Should raise a lint error.

## Environment
 - Version 4.2.4
 - Platform RHEL8
